Classes/New Issue/IssueTemplateHelper.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  IssueTemplates.swift
+//  Freetime
+//
+//  Created by Ehud Adler on 11/3/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+import GitHubAPI
+import GitHubSession
+import Squawk
+
+struct IssueTemplate {
+    let title: String
+    let template: String
+}
+
+struct IssueTemplateDetails {
+    let repo: RepositoryDetails
+    let session: GitHubUserSession?
+    let viewController: UIViewController
+    weak var delegate: NewIssueTableViewControllerDelegate?
+}
+
+class IssueTemplateHelper {
+
+    static func getNameAndDescription(fromTemplatefile file: String) -> (name: String?, about: String?) {
+        let names = file.matches(regex: Constants.Regex.Template.name)
+        let abouts = file.matches(regex: Constants.Regex.Template.about)
+        let name = names.first?.trimmingCharacters(in: .whitespaces)
+        let about = abouts.first?.trimmingCharacters(in: .whitespaces)
+        return (name, about)
+    }
+
+    static func showIssueAlert(
+        with templates: [IssueTemplate],
+        details: IssueTemplateDetails) -> UIAlertController {
+
+        let alertView = UIAlertController(
+            title: NSLocalizedString("New Issue", comment: ""),
+            message: NSLocalizedString("Choose Template", comment: ""),
+            preferredStyle: .alert
```

Comment:
I'm thinking action sheet might be better here, but this is personal preference

Reply:
Bas also mentioned this, I'll switch for now and we can switch back if need be